packages/data/src/namespace-store/test/index.js

```diff
@@ -74,17 +74,16 @@ describe( 'controls', () => {
 			},
 		} );
 
-		registry.subscribe( () => {
+		registry.subscribe( async () => {
```

Comment:
I don't think this is quite the same as it was previously. I believe the reason we'd used `done` is because we couldn't be certain when the `subscribe` callback would be invoked. Changing the callback to `async` won't help this. As with my earlier comment, I think if we need to convert this to a promise, we can `return new Promise` from the test case and attach the `subscribe` within, then call `resolve()` in the same place we'd called `done`.

Reply:
It makes sense, I will do it 👍